widget/embedded/src/store/quote.ts

```diff
@@ -289,3 +297,58 @@ export const useQuoteStore = createSelectors(
     }))
   )
 );
+
+// Unsubscribe listeners
+export const unsubscribeQuoteStore = useQuoteStore.subscribe(
```

Comment:
Are you unsubscribing here? It seems it's subscribing code.

Reply:
https://github.com/rango-exchange/rango-client/pull/763#discussion_r1645344282